app/src/main/java/com/codesoom/assignment/handlers/TaskHttpHandler.java

```diff
@@ -0,0 +1,173 @@
+package com.codesoom.assignment.handlers;
+
+import com.codesoom.assignment.http.HttpMethod;
+import com.codesoom.assignment.http.HttpStatus;
+import com.codesoom.assignment.models.ResponseDto;
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.models.TaskDto;
+import com.codesoom.assignment.services.TaskService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.Headers;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class TaskHttpHandler implements HttpHandler {
+
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+    private final TaskService taskService;
+
+    private static final String PATH = "/tasks";
+    private static final String PATH_REGEX = "^\\/tasks\\/[0-9]+$";
+    private static final Pattern PATH_PATTERN = Pattern.compile(PATH_REGEX);
+
+    public TaskHttpHandler() {
+        this.taskService = new TaskService();
+    }
+
+    /**
+     * /tasks/{id} 형식에 맞는 URI라면 true를 반환한다.
+     */
+    private boolean isMatch(String requestURI) {
+        final Matcher matcher = PATH_PATTERN.matcher(requestURI);
+        return matcher.matches();
+    }
```

Comment:
나중에 URI를 바꾸게 되었을 때 주석까지 수정하게 되면 귀찮겠죠. 객체지향 원칙은 주석에도 통합니다.
```suggestion
    /**
     * 할 일 조회 형식에 맞는 URI라면 true를 반환한다.
     */
    private boolean isMatch(String requestURI) {
        final Matcher matcher = PATH_PATTERN.matcher(requestURI);
        return matcher.matches();
    }
```